### ISSUE TITLE
Define how QuickStarts can be surfaced on the Dev Site in Virtouso and NR1

### ISSUE THREAD

**Issue by jpvajda** [newrelic/developer-website#1211]
We are wanting to turn this https://newrelic.github.io/nr1-quickstarts/#/ into a proper section on the developer site, so it's easy to use, maintain and follows are standards for developer how tos and guides.

Quick starts may eventually be added to the Virtuoso install workflow, so there will be some desire to unify this expeirence with whatever is introduced to that flow. 

Finally, it would make sense to surfacing in the NR1 Product as currently they are found under 3rd party apps which isn't so intuitive. 

### Design Ideas & Tasks

1. Add a primary nav item Quick Starts
2. Decide on new icon for Quick starts
3. Follow the pattern established by [Build Apps](https://developer.newrelic.com/build-apps) which has 

- a intro paragraph
- a intro video
- tiles that highlight each quick start item and link to the quick start repositories
- We will probably want an index page that uses this content [found here](https://newrelic.github.io/quickstarts-dashboard-library/#/installation)

### Additional info & feedback

There is a listing quickstarts [here](https://newrelic.github.io/quickstarts-dashboard-library/#/) and instructions on how to set these up [here](https://newrelic.github.io/quickstarts-dashboard-library/#/installation). This uses a [3rd party app](https://one.newrelic.com/launcher/catalog.launcher?launcher=eyJhY3RpdmVWaWV3IjoiZGV0YWlscyIsInBhY2thZ2VJZCI6ImNkMmZjMjg2LWViMTUtNDQzYS1hMDlmLWYwOGI1OTkzNzk2ZCIsInBhY2thZ2VOYW1lIjoiUXVpY2tzdGFydHMifQ==&pane=eyJuZXJkbGV0SWQiOiJjYXRhbG9nLmhvbWUiLCJhY3RpdmVWaWV3IjoiZGV0YWlscyIsInBhY2thZ2VJZCI6ImNkMmZjMjg2LWViMTUtNDQzYS1hMDlmLWYwOGI1OTkzNzk2ZCIsInBhY2thZ2VOYW1lIjoiUXVpY2tzdGFydHMifQ==&platform[accountId]=) you add to the catalog (which doesn't work on my test account) 🤷 

So it's very disjointed at the moment and it's really hard to follow what we are expecting user to do here.

### possible Virtuoso workflow

1. run the installer in the UI or CLI 
2. make all your installer selections
2. pick your pre-built dashboards from a list of available ones
3. Get linked into to the product. 
4. find New Relic Joy. 

`Guided Install` eventually if you want to use our sandbox to replicate fake production data you can, instead of installing on your own infrastructure.

### other ideas 

when a user clicks on create a dashboard or import a dashboard in the NR1 Product, why not let them select a quickstart dashboard?

<img width="402" alt="Screen Shot 2021-03-31 at 2 48 07 PM" src="https://user-images.githubusercontent.com/1395158/113209549-71c8c280-9230-11eb-98fb-38d07d99a417.png">

**Import**
<img width="425" alt="Screen Shot 2021-03-31 at 2 48 15 PM" src="https://user-images.githubusercontent.com/1395158/113209589-7c835780-9230-11eb-847d-b12b0dec1153.png">

**Create**
<img width="463" alt="Screen Shot 2021-03-31 at 2 47 22 PM" src="https://user-images.githubusercontent.com/1395158/113209639-8c9b3700-9230-11eb-9928-2b8806b34dab.png">

**Comment by jpvajda** [comment]
@polfliet here is my assessment on this let me know what we want to do next. 🚀

**Comment by polfliet** [comment]
One assumption (that needs to be verified if that’s in the scope of the Virtuoso team):
* Quickstarts will be part of the install recipe (see example [here](https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/infrastructure/ohi/mysql/debian.yml))
* This means there is a static GitHub link baked into the install recipe itself.
* This way, a user automatically gets a recommended quickstart installed.
* This means that for the initial in-product XP I suggest we call out what quickstart was deployed, and potentially draw attention how more quickstarts can be deployed (by refering to the quickstarts NR1 app, which should become native) or customized/contributed (by refering to the GitHub repo)
This makes the initial in-product XP a lot simpler. As part of the New Relic Sandbox work, we can build 1 great quickstart, but we will need to support from the community to contribute more and build a true ecosystem.

About your other suggestion `when a user clicks on create a dashboard or import a dashboard in the NR1 Product, why not let them select a quickstart dashboard?` I believe this is on the roadmap for the dashboard team, but will check.

**Comment by jpvajda** [comment]
we've done this work in our requirments doc, so closing this issue.
